assets/js/romo/base.js

```diff
@@ -671,49 +671,50 @@ Romo.prototype._addEventCallback = function(name, callback) {
   this._eventCallbacks.push({ eventName: name, callback:  callback });
 }
 
-// RomoParentChildElems (TODO: rework w/o jQuery)
+// RomoParentChildElems
 
 var RomoParentChildElems = function() {
   this.attrName = 'romo-parent-elem-id';
   this.elemId   = 0;
   this.elems    = {};
 
-  var parentRemovedObserver = new MutationObserver($.proxy(function(mutationRecords) {
-    mutationRecords.forEach($.proxy(function(mutationRecord) {
+  var parentRemovedObserver = new MutationObserver(Romo.proxy(function(mutationRecords) {
+    mutationRecords.forEach(Romo.proxy(function(mutationRecord) {
       if (mutationRecord.type === 'childList' && mutationRecord.removedNodes.length > 0) {
-        $.each($(mutationRecord.removedNodes), $.proxy(function(idx, node) {
-          this.remove($(node));
+        mutationRecord.removedNodes.forEach(Romo.proxy(function(removedNodeElem) {
+          this.remove(removedNodeElem);
         }, this));
       }
     }, this));
   }, this));
 
-  parentRemovedObserver.observe($('body')[0], {
+  parentRemovedObserver.observe(Romo.f('body')[0], {
     childList: true,
     subtree:   true
   });
 }
 
 RomoParentChildElems.prototype.add = function(parentElem, childElems) {
-  parentElem.attr('data-'+this.attrName, this._push(childElems, parentElem.data(this.attrName)));
+  Romo.attr(parentElem, 'data-'+this.attrName, this._push(childElems, Romo.data(parentElem, this.attrName)));
 }
 
 RomoParentChildElems.prototype.remove = function(nodeElem) {
-  if (nodeElem.data('romo-parent-removed-observer-disabled') !== true) {
-    if (nodeElem.data(this.attrName) !== undefined) {
-      this._removeChildElems(nodeElem);  // node is a parent elem itself
+  if (Romo.data(nodeElem, 'romo-parent-removed-observer-disabled') !== true) {
+    if (Romo.data(nodeElem, this.attrName) !== undefined) {
```

Comment:
@kellyredding - I think the `Romo.data` returns `null` when the data attribute isn't on the element (that's what I see when I test it in the console).

I saw these in the components and meant to ask if we wanted to change the `Romo.data` to return `undefined` instead but forgot about it until I saw this change.

Reply:
@jcredding I addressed this in https://github.com/redding/romo/pull/143, FYI